serial.ts

```diff
@@ -35,29 +49,30 @@ const kAcceptableDataBits = [16, 8, 7, 6, 5];
 const kAcceptableStopBits = [1, 2];
 const kAcceptableParity = ['none', 'even', 'mark', 'odd', 'space'];
 
-const kParityIndexMapping = ['none', 'odd', 'even', 'mark', 'space'];
+const kParityIndexMapping: ParityType[] =
+    ['none', 'odd', 'even', 'mark', 'space'];
 const kStopbitsIndexMapping = [1, 1.5, 2];
 
 /** a class used to control serial devices over WebUSB */
-class SerialPort {
+export class SerialPort {
+  public in: ReadableStream<Uint8Array>;
```

Comment:
Any particular reason you chose the arraybuffer to be typed as Uint8Array?  Am I correct in assuming the actual serial implementation will type this as Uint8Array as well?

Reply:
This indicates that the stream is a byte stream (ArrayBuffer is much more generic). This is the type that will be used by the Serial API specification. It is also required if we want to support a BYOB (bring your own buffer) stream when those are eventually implemented in Chrome.